### PR TITLE
fix(handlers/channel): handle max_age and max_uses limit in createInvite()

### LINKED ISSUE #580: Support min max limits for creating invites
https://github.com/discord/discord-api-docs/commit/ddca46a3cfb5536ab17bbfa1756d89769580d708

### PR DESCRIPTION
**Pull request by Skillz4Killz**
Closes #580

### CODE REVIEW

File: src/api/handlers/channel.ts [hunk 1]
```diff
@@ -321,6 +321,24 @@ export async function createInvite(
...
+    );
+    if (options.max_age > 604800 || options.max_age < 0) {
+      options.max_age = undefined;
+    }
+  }
+
+  if (options.max_uses) {
+    console.log(
+      `The max uses for invite created in ${channelID} was invalid. Using default values instead.`,
+    );
```
Comment (ayntee) [inline src/api/handlers/channel.ts:336]: If it is provided, it is invalid? Strange, can you clarify this?

File: src/api/handlers/channel.ts [hunk 2]
```diff
@@ -321,6 +321,24 @@ export async function createInvite(
...
   }
 
+  if (options.max_age) {
+    console.log(
+      `The max age for invite created in ${channelID} was invalid. Using default values instead.`,
+    );
+    if (options.max_age > 604800 || options.max_age < 0) {
+      options.max_age = undefined;
+    }
+  }
```
Comment (itohatweb) [inline src/api/handlers/channel.ts:331]:
```suggestion
  if (options.max_age > 604800 || options.max_age < 0) {
    console.log(
      `The max age for invite created in ${channelID} was invalid. Using default values instead.`,
    );
    options.max_age = undefined;
  }
```

File: src/api/handlers/channel.ts [hunk 3]
```diff
@@ -321,6 +321,24 @@ export async function createInvite(
...
+
+  if (options.max_uses) {
+    console.log(
+      `The max uses for invite created in ${channelID} was invalid. Using default values instead.`,
+    );
+    if (options.max_uses > 100 || options.max_uses < 0) {
+      options.max_uses = undefined;
+    }
+  }
```
Comment (itohatweb) [inline src/api/handlers/channel.ts:340]:
```suggestion
  if (options.max_uses > 100 || options.max_uses < 0) {
    console.log(
      `The max uses for invite created in ${channelID} was invalid. Using default values instead.`,
    );
    options.max_uses = undefined;
  }
```

**Review by itohatweb — CHANGES_REQUESTED** [review]

**Review by itohatweb — APPROVED** [review]

**Review by ayntee — APPROVED** [review]
LGTM

**Pull request merged**